docs/bots-guide.md

```diff
@@ -76,12 +76,16 @@ You need:
 **Note: Please be considerate when testing experimental bots on
   public servers such as chat.zulip.org.**
 
-1. Install all requirements. You can either
-
-    * run `pip install zulip_bots` for a stable version, or
-    * clone the [`zulip_bots`](https://github.com/zulip/python-zulip-api/tree/master/zulip_bots)
-      repository for the latest code. Install it with
-      `pip -e <path/to/zulip_bots>`; you will be able to make changes to the code.
+1. Install all requirements. You can **either**
+
+    * run `pip install zulip_bots` for a stable version, **or**
+    * install the latest version from our repository:
+      - `git clone https://github.com/zulip/python-zulip-api.git` - clone the [python-zulip-api](
+        https://github.com/zulip/python-zulip-api) repository.
+      - `cd python-zulip-api` - navigate into your cloned repository.
+      - `pip install ./zulip_bots` - install the `zulip_bots` package.</br>
```

Comment:
Should this just document `./tools/provision` instead?

Reply:
Yes, we should probably do that. My thinking was that we should only need the `zulip_bots` package to be installed, but then again, an Outreachy participant reported dependency issues when running this command, so provision is better here.